### ISSUE TITLE
Email: Comments on WCAG 3.0 draft (13 - Recommendations)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#492]
[Comment from Email](https://lists.w3.org/Archives/Public/public-silver/2021Mar/0002.html): 
from [Word docx](https://lists.w3.org/Archives/Public/public-silver/2021Mar/att-0002/Review_Comments_for_WCAG_3.0_-_Vanderheiden.docx) 

Recommendations – possible solutions

1.	Rename the document and shift it to being a guidance document
a.	the easiest way to preserve all of the value of this document is to rename it so that it is not trying to be a document in the WCAG line  --  that is developed for use by people wanting objective standard that could be used in regulatory or other judgmental or criterion based evaluations ( e.g. as a requirements documents for purchasing or for use inside a company, inside an organization, inside a country, or internationally.)
b.	Such a document could be very valuable as a basis for continuous quality improvement for accessibility.

2.	Alternatively, one could decide to target it as an objective standard that could be used for in regulatory or other requirements use
a.	But this would means having to follow the current WCAG model for selecting normative provisions  (see below) 
b.	This would allow the document to be used as a continuation in a WCAG line but would defeat many of the goals of the group putting this document together.  
c.	The criteria for inclusion of provisions in WCAG 2.2  (from the working group wiki) are as follows:   
Success Criteria shall: 
1.	Address a situation where a user with a disability will be disproportionately disadvantaged (as compared to a user without a disability) if the criterion is not met.
2.	Be testable through an automated process or by a manual process conducted by an individual evaluator, and any tools required to test it are available prior to Candidate Recommendation stage.
3.	Provide consistent results from different testers (e.g. 8/10 testers agree). This can be assessed through inherent logic or proven through examples.
4.	Describe the specific condition required to meet the criteria, not the method to address the criteria.
5.	Utilize the WCAG 2.x A/AA/AAA level structure.
6.	Apply to all content across all websites unless preconditions for the application of the success criteria are explicitly identified (e.g. "except interruptions involving an emergency").
7.	Apply across technologies to the greatest extent possible. (Technology-specific issues should usually be addressed in Techniques.)
8.	Avoid creating a requirement for something that is already required by an existing Success Criterion.
Success Criteria should: 
1.	Be as broad as possible, but specific enough not to become a 'catch-all' for any given requirement.
2.	Not require testing that is believed to require very large manual efforts. If a tool is very likely to be available soon after publication that makes the testing more efficient, this factor is less important.
3.	Use glossary definitions to simplify and shorten all Success Criteria for shared or ambiguous terms.

3.	Since the guidelines are targeted to 3 different audiences (authors, user agent developers, and authoring tool developers) either:
a.	Break the provisions into 3 sections – with each audience having a section they follow
OR
b.	Clearly separate provisions meant for one audience from provisions meant for another and tag or label them as such
i.	So that each group and find what applies to them
ii.	So that each group can meet the relevant provisions because they do not contain parts that are outside of their control 

4.	Ensure that no provision is included until the working group goes and samples a good cross-section of the major sites, and the working group itself is able to rewrite them to show how they would/could conform.   Implementations on at least 3 very different sites should be done for each and for all of the provisions (all the provisions applied to a meaningful number of pages representing the different page types for each of 3 sites).
a.	This can prevent provisions that are not implementable from showing up in the guidelines 
b.	It can help identify places where exceptions or additional conditions or allowances are needed  (in WCAG 2 we learned much from this exercise.) 
If the working group can't figure out how to redo the sites to comply – then it is likely that it is beyond the abilities of most web developers.  And if it is too much work to do this for that limited number of pages – by the experts in the working group, then it is likely too hard to do.
5.	Continue with your Outcome-oriented approach.
a.	Provisions should state what is desired – rather than what the content, tool, or agent author should do. 
b.	How to do it should be in separate document that can be updated as technologies change.  Not only how, but even ‘what’ needs to be done will change over time. 
c.	This allows the provisions to remain stable – which is a key requirement for inclusion in regulations that may not change for 5 or 10 years sometimes. 

6.	Use a consistent, straightforward and labelled format.
a.	The three examples do not have exactly the same format.  Also, they have sections that seem to be repetitive in functions sometimes.
b.	Suggested format  {if you are going for a testable guidelines.  If not then you can ignore the “normative/ informative” labels below – but this might still be a good format.)
i.	Short name for provision 
1.	(makes it easy to refer to them in conversation and to find the one you are looking for
ii.	Provision 
1.	(In bold or highlighted or boxed text)
2.	This is the normative part of the provision
3.	Shortname can be in front of or above the provision
4.	The provision is in outcome format
5.	The provision does not refer to users or what users can or should be able to do – but rather what the should be true about the content, agent, or tool
iii.	A Rationale or benefit statement.   
1.	This is non-normative, informative 
2.	It is below the provision and is labelled. 
a.	To follow standard format for standards it probably should be labelled as  NOTE:
3.	This note can use the word users and explains the purpose or benefit of the provision.
a.	This would help in understanding why the provision is needed.
b.	It can help readers to understand the different groups that can benefit from a provision (though that should be done in more detail in a separate document that discusses each provision in detail)
4.	Examples:
a.	NOTE: This allows users who are unable to perceive and / or understand the non-text content to determine its meaning.
b.	NOTE: Text alternatives to audio content allows users to access the information in the audio when they cannot hear the content or hear it clearly (e.g. if they are hard of hearing or deaf, or if it is too noisy for them to understand).  Having the text alternative be accessible to assistive technologies allows it to also be translated into other languages, or into symbols, braille or other forms now or in the near future. 
c.	NOTE: What are called captions in some countries are called subtitles in others – even if in the same language as the audio.
iv.	A link to further resources
1.	This allows users who are unable to perceive and / or understand the non-text content to determine its meaning

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  The Accessibility Guidelines Working Group attempted to address these concerns in this latest draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
